exercises/hello-world/lib/example.dart

```diff
@@ -1,3 +1,3 @@
 class HelloWorld {
-  String hello([String name = "World"]) => "Hello, $name!";
+  String hello() => "Hello, World!";
```

Comment:
I recommend changing the fat arrow (=>) to a block code. Beginners would want to know what exactly is happening here.

Reply:
Explicitly adding the return statement would prove helpful to new comers and make them feel familiar to the syntax if they come form a mainstream language such as c++, Java.

Reply:
@kabiir I take it you are in favor of this change which is being discussed in #57 ?

Reply:
Yeah, hello world is the kind of exercise which beginners should just be able to get past through it easily. They'll learn about parameters in proceeding exercises.

Traditionally *hello world* is about how to get output in a particular language. Which is using print() in dart but the test requires the output as a return and not as an actual output on the console. So this will do.